.github/workflows/release.yaml

```
@@ -47,6 +49,9 @@ jobs:
       - name: Kubernetes ${{ matrix.kubernetes_version }} ${{ matrix.dbmode }} Integration Tests
         run: KONG_CLUSTER_VERSION=${{ matrix.kubernetes_version }} make test.integration.${{ matrix.dbmode }}
 
+      - name: Kubernetes ${{ matrix.kubernetes_version }} ${{ matrix.enterprise }} Integration Tests
+        run: KONG_CLUSTER_VERSION=${{ matrix.kubernetes_version }} make test.integration.enterprise
+
```

Seems like we can drop the matrix fields since we're not using them, and also based on the makefile it looks like we need to add the dbmode:

```suggestion
      - name: Kubernetes ${{ matrix.kubernetes_version }} Kong Enterprise Integration Tests
        run: KONG_CLUSTER_VERSION=${{ matrix.kubernetes_version }} make test.integration.enterprise/${{ matrix.dbmode }}

```

---

https://github.com/Kong/kubernetes-ingress-controller/pull/1758/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR196